sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/intervalExpressions.scala

```diff
@@ -345,6 +345,109 @@ case class MakeInterval(
     )
 }
 
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage = "_FUNC_(days, hours, mins, secs) - Make DayTimeIntervalType duration from days, hours, mins and secs.",
+  arguments = """
+    Arguments:
+      * days - the number of days, positive or negative
+      * hours - the number of hours, positive or negative
+      * mins - the number of minutes, positive or negative
+      * secs - the number of seconds with the fractional part in microsecond precision.
+  """,
+  examples = """
+    Examples:
+      > SELECT _FUNC_(1, 12, 30, 01.001001);
+       1 12:30:01.001001000
+      > SELECT _FUNC_(100, null, 3);
+       NULL
+  """,
+  since = "3.2.0",
+  group = "datetime_funcs")
+// scalastyle:on line.size.limit
+case class MakeDuration(
```

Comment:
MakeDTInterval?

Reply:
Well, I think the name "Duration" also keeps consistency with [Java Duration](https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html), which is more comprehensive by common users.
By the way, it's not easy to associate the abbreviation of "DT"  with "Day Time" for common users.

Reply:
As the conversation in https://github.com/apache/spark/pull/32645, I will rename to MakeDTInterval.